applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py

```diff
@@ -29,6 +59,11 @@ def __init__(self, grid_model_part, initial_material_model_part, material_model_
         # Default settings string in json format
         default_settings = KratosMultiphysics.Parameters("""
         {
+            "model_part_name" : "MPM_Material",
+            "domain_size"     : 2,
+            "time_stepping" : {
+                "time_step"       : 0.0005
```

Comment:
I think you should remove this => this way the user gets notified in case it is not specified